charts/prometheus-rabbitmq-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Rabbitmq metrics exporter for prometheus
 name: prometheus-rabbitmq-exporter
-version: 0.6.0
+version: 0.7.0
```

Comment:
Probably a breaking change, since it changes/deletes rules. Better be safe!

Reply:
yep